CHANGELOG.md

```diff
@@ -1,5 +1,445 @@
 # Changelog
 
+## [1.13.0] (2024-07-09)
+
+[1.13.0]: https://github.com/crystal-lang/crystal/releases/1.13.0
+
+### Features
+
+#### lang
+
+- Allow rescuing exceptions that include a module ([#14553], thanks @Blacksmoke16)
+- *(macros)* Allow assignment to `_` inside macro expressions ([#14452], thanks @HertzDevil)
+
+[#14553]: https://github.com/crystal-lang/crystal/pull/14553
+[#14452]: https://github.com/crystal-lang/crystal/pull/14452
+
+#### stdlib
+
+- *(collection)* Add `Array#insert_all` ([#14486], thanks @summer-alice)
+- *(collection)* Improve compile time error for `#sort(&block : T, T -> U)` ([#14693], thanks @beta-ziliani)
+- *(concurrency)* Add `WaitGroup` synchronization primitive ([#14167], thanks @ysbaddaden)
+- *(concurrency)* Add `Thread.new` yields itself ([#14543], thanks @ysbaddaden)
+- *(concurrency)* Add support for `Atomic(Bool)` ([#14532], thanks @ysbaddaden)
+- *(concurrency)* Add `Thread.sleep(Time::Span)` ([#14715], thanks @ysbaddaden)
+- *(files)* Implement `IO#tty?` in Win32 ([#14421], thanks @HertzDevil)
+- *(files)* Implement `File.readable?` and `.writable?` in Win32 ([#14420], thanks @HertzDevil)
+- *(files)* Make `File.readable?` and `.writable?` follow symlinks on Windows ([#14514], thanks @HertzDevil)
+- *(llvm)* Add some missing `LLVM::Context` bindings ([#14612], thanks @ysbaddaden)
+- *(llvm)* Do not strip the macOS target triple ([#14466], thanks @hovsater)
+- *(log)* Support `UInt32` and `UInt64` in Log Context ([#14459], thanks @toddsundsted)
+- *(macros)* Add AST node methods for macro-related nodes ([#14492], thanks @HertzDevil)
+- *(macros)* Support short blocks in `#[]` operator call's macro interpolation ([#14523], thanks @HertzDevil)
+- *(macros)* Add macro methods for `Select` ([#14600], thanks @HertzDevil)
+- *(macros)* Add `TypeNode#private?`, `#public?` and `#visibility` ([#11696], thanks @Hadeweka)
+- *(macros)* Add `StringLiteral#to_utf16` ([#14676], thanks @ysbaddaden)
+- *(networking)* Relax type restriction of handlers in `HTTP::Server.new` to `Indexable(HTTP::Handler)` ([#14413], thanks @hugopl)
+- *(networking)* **[security]** OpenSSL: don't change default cipher suites ([#14655], thanks @ysbaddaden)
+- *(networking)* Allow parsing cookies with space in the value ([#14455], thanks @anton7c3)
+- *(runtime)* Allow `Atomic`s of pointer types ([#14401], thanks @HertzDevil)
```

Comment:
**question**: shouldn't `Atomic(Bool)` and `Atomic(Pointer)` be grouped together? That is both be `concurrency` or `runtime`.